frontend/src/routes/CriticalQuestions/index.js

```diff
@@ -259,11 +277,14 @@ function CriticalQuestions(props) {
         <div key={i} className={styles.contact}>
           <div className={classNames("grid-3", styles["grid-3"])}>
             <TextField
+              error={contact.email && !validateEmail(contact.email)}
               label="Email *"
               value={contact.email}
               onChange={handleCloseContactChange("email", i)}
+              helperText="Please enter a valid email."
```

Comment:
There is a line of black text here. It should not be shown if the user does not input anything yet. 
![image](https://user-images.githubusercontent.com/37783082/89865292-4f2f4f80-db62-11ea-8475-59b37f4899e6.png)

Reply:
Would that be better? I intentionally left in like this because I don't think it's good to show so much red when user click the add.

Reply:
Like it's similar to how we only remind user to enter required field when they try to move forward without filling out those information. And here we only want to remind user to enter valid email when they entered invalid information.

Reply:
But it is black, the user cannot even see it. When I am typing, the warning message just changes from black to red.

Reply:
![Screen Shot 2020-08-12 at 09 24 34](https://user-images.githubusercontent.com/37783082/90041080-a4ac4f00-dc7d-11ea-8a15-7e6d45e9aa3a.png)
Duplicate error message. I think keep the one under email field is good.

Reply:
Actually one my end, if you scroll all the way up to enter the city, you might not actually see the message down there. So I think keeping it there might help user realize what they are missing.

Reply:
Ok. We will keep both.